nixos/roles/ceph/mon.nix

```diff
@@ -285,6 +289,35 @@ in
         };
       };
     })
+    (lib.mkIf (role.enable && fclib.ceph.releaseAtLeast "nautilus" role.cephRelease ) {
+      flyingcircus.passwordlessSudoRules = [
+        {
+          commands = [
+            "${fc-check-ceph-withVersion}/bin/check_snapshot_restore_fill"
+          ];
+          groups = [ "sensuclient" ];
+        }
+      ];
+
+      flyingcircus.services.sensu-client.checks = let
+          # check config generated directly from our platform settings
+          configtoml = (pkgs.formats.toml {}).generate "config.toml" {
+            thresholds = {
+              # use canonical, non-camelCase form of ceph settings
+              nearfull = config.flyingcircus.services.ceph.allMergedSettings.mon."mon osd nearfull ratio";
+              full = config.flyingcircus.services.ceph.allMergedSettings.mon."mon osd full ratio";
+            };
+            ceph_roots = config.flyingcircus.services.ceph.server.root_rbdpool_mapping;
+          };
+        in {
+          ceph_snapshot_restore_fill = {
+            notification = "There is enough space to be able to restore the largest " +
```

Comment:
Notification should describe exactly the opposite condition, i.e. what to neotify when the check actually triggers.